qualtran/bloqs/arithmetic/Numerical analysis of error propagation due to fixed point arithmetic.ipynb

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
Are we ok with spaces in filenames?

Reply:
Renamed to remove spaces and match pattern used by other notebooks